docs/proposals/odo-v3-cli/odo-dev.md

```diff
@@ -0,0 +1,91 @@
+# `odo dev`
+
+[#5299](https://github.com/redhat-developer/odo/issues/5298)
+
+Running the application on the cluster for **development** (inner loop)
+
+The result of successful execution of `odo dev` is Devfile component deployed in inner loop mode.
+**By default command deletes all resources that it created before it exits.**
```

Comment:
Why? And what resources exactly? Does it also delete the deployments that were created for this devfile component, or just the URL, storages, services that were created while `odo dev` was executing?
And what happens to URL, storages, and services that were present when `odo dev` began executing? Will those be deleted as well?
And if we also delete deployments, then what is the point of having `odo list` for component's pushed via `odo dev`?
Or will all of this make sense when we implement `--cleanup` flag?